src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/ProfessionalExternalUserController.java

```diff
@@ -112,17 +112,14 @@
 
         profExtUsrReqValidator.validateRequest(organisationIdentifier, showDeleted, status);
 
-        ResponseEntity<Object> profUsersEntityResponse;
-
         if (!organisationIdentifierValidatorImpl.ifUserRoleExists(jwtGrantedAuthoritiesConverter.getUserInfo()
                 .getRoles(), PUI_USER_MANAGER)) {
-            status = isBlank(status) ? ACTIVE : status;
+            status = StringUtils.isEmpty(status) ? ACTIVE : status;
```

Comment:
I think you need to use `StringUtils.isBlank(status)` so that if the status is `"  "` which has an extra space then it would be default to `ACTIVE`

Reply:
@SujeethKumarAri done.